### ISSUE TITLE
Model optimization

### ISSUE THREAD

**Issue by MM-DHM** [MIC-DKFZ/nnUNet#1791]
Dear Fabian,

thanks a lot for your work. I am curerntly running nnUNet on 3D images of Coronary CTA. But there is not that mutch of improvement after 100 Epochs, do you have any sugessions?

![Unbenannt](https://github.com/MIC-DKFZ/nnUNet/assets/124573108/0153c56d-f720-4d0b-bb62-6735119d6932)

**Comment by Kobalt93** [comment]
Hi MM-DHM,
It is true that the loss does not decrease in such a peed as before, but it seems for me that the validation loss follows your training loss really nicely, which means that you are not overfitting and your model still learns important features. The question is that dice score of ~0.78 is good enough for your applcation and whether more could be reached. I would consider the followings based on your specific dataset:
- Using another loss function for thin structures e.g.: https://neurips.cc/virtual/2023/82494, https://arxiv.org/pdf/2003.07311.pdf
- Use data-specific data augmentations to boost your performance, e.g.: https://arxiv.org/pdf/2309.03652.pdf
